src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -1145,7 +1112,7 @@ describe('MenuBar', () => {
   template: `
     <ul cdkMenuBar>
       <li role="none">
-        <button checked="true" cdkMenuItemRadio>
+        <button cdkMenuItemChecked="true" cdkMenuItemRadio>
```

Comment:
I see the appeal because its less verbose, though if we do that there are some examples that feel weird:
- `<div cdkMenuItemRadio>` - is checked, as a user I would probably expect unchecked
- `<div cdkMenuItemRadio="false">` - sounds it should make it behave like not a radio button or something

Reply:
Yeah that's also a good point, especially if we end up coercing the input. Up to you if you want to change it.